### PR TITLE
Refactor: Update default nav template

### PR DESCRIPTION
**Pull request by erikjung**
Re: #23 
### Overview

This makes a variety of changes to the default patterns, pages, and navigation.
- A few more patterns have been added ("Elements") to better demonstrate that you can break patterns into multiple sections (aside from "Components")
- The color swatches have been placed in a new standalone page
- Metadata has been added to affect the sort order of some collections
- The navigation template has been updated with headings for each grouping, and now uses the sorting feature (https://github.com/cloudfour/drizzle-builder/pull/95) of the `{{collections}}` helper to control the order or collections.

---

screen shot 2016-05-25 at 3 25 00 pm

---

screen shot 2016-05-25 at 3 25 17 pm

---

/CC @mrgerardorodriguez @saralohr @nicolemors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "babel-preset-es2015": "^6.6.0",
     "core-gulp-tasks": "github:cloudfour/core-gulp-tasks#v2.0.0",
     "core-hbs-helpers": "github:cloudfour/core-hbs-helpers#v0.4.0",
-    "drizzle-builder": "0.0.7",
+    "drizzle-builder": "0.0.9",
```
Comment (tylersticka) [inline package.json:35]: Racking up those patch releases! :trollface:

Comment (erikjung) [inline package.json:35]: Yeah, the impending _minor_ release. Realness.

**Comment by tylersticka** [comment]
LGTM :+1:

**Comment by gerardo-rodriguez** [comment]
Yay! 👍

**Pull request merged**